{{cookiecutter.python_name}}/conftest.py

```diff
@@ -0,0 +1,8 @@
+import pytest
+
+pytest_plugins = ["jupyter_server.pytest_plugin"]
```

Comment:
i thought the newer pytest started warning about doing this in a config (e.g. pyproject.toml, setup.cfg, tox.ini or pytest.ini)

Reply:
This is the way documented in the [official documentation](https://docs.pytest.org/en/7.0.x/how-to/plugins.html#requiring-loading-plugins-in-a-test-module-or-conftest-file)